website/server/controllers/api-v3/chat.js

```diff
@@ -276,6 +276,7 @@ api.likeChat = {
  *
  * @apiParam (Path) {UUID} groupId The group id ('party' for the user party and 'habitrpg' for tavern are accepted)
  * @apiParam (Path) {UUID} chatId The chat message id
+ * @apiParam (Body) {String} comment The text of the message (optional)
```

Comment:
thanks @Juliusdotsh !

Two quick things:

- to mark the field as optional instead of writing `optional` you can just put the field name in square brackets and it'll be shown as optional so `comment` -> `[comment]`
- can you change `The text of the message` to `A comment to explain why the message was flagged`

Reply:
@paglias I've fixed the issue but i think i created more harm then god considering git counts five commits when i made only one, I still don't know what happened

Reply:
@Juliusdotsh it happens sometimes when merging new commits from the `develop` branch. My suggestion is to create a new clean branch and a new PR from there

Reply:
#10925